[
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     834     gcc: _configtest.c\n835     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     836         3 | #pragma clang diagnostic error \"-Wattributes\"\n     837           |\n     838     _configtest.c: In function '__declspec':\n  >> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n  >> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {\n     844           | ^\n845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n  >> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |",
    "implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n  >> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |\n851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |\n     854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "849        11 | }\n     850           |\n     851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |\n854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"\n     856           |\n     857     _configtest.c: In function '__declspec':\n  >> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n  >> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {\n     863           | ^\n864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n  >> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |",
    "implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n  >> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |\n870     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     871        11 | }\n     872           |\n     873     failure.\n\n     ...",
    "...\n\n     2948          |                              ^~\n     2949    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2950    numpy/core/src/multiarray/scalartypes.c.src:2754:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2951     2754 |         if (ret->obmeta.base == -1) {\n     2952          |                              ^~\n     2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2974          |                              ^\n     2975    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2976    numpy/core/src/multiarray/scalartypes.c.src:637:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2977      637 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2978          |     ^~\n\n     ...",
    "...\n\n     3115          |                              ^~\n     3116    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3117    numpy/core/src/multiarray/scalartypes.c.src:2754:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3118     2754 |         if (ret->obmeta.base == -1) {\n     3119          |                              ^~\n     3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     834     gcc: _configtest.c\n835     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     836         3 | #pragma clang diagnostic error \"-Wattributes\"\n     837           |\n     838     _configtest.c: In function '__declspec':\n  >> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n  >> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {\n     844           | ^\n845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n  >> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |",
    "implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n  >> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |\n851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |\n     854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "849        11 | }\n     850           |\n     851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |\n854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"\n     856           |\n     857     _configtest.c: In function '__declspec':\n  >> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n  >> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {\n     863           | ^\n864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n  >> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |",
    "implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n  >> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |\n870     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     871        11 | }\n     872           |\n     873     failure.\n\n     ...",
    "...\n\n     2948          |                              ^~\n     2949    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2950    numpy/core/src/multiarray/scalartypes.c.src:2754:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2951     2754 |         if (ret->obmeta.base == -1) {\n     2952          |                              ^~\n     2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2974          |                              ^\n     2975    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2976    numpy/core/src/multiarray/scalartypes.c.src:637:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2977      637 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2978          |     ^~\n\n     ...",
    "...\n\n     3115          |                              ^~\n     3116    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3117    numpy/core/src/multiarray/scalartypes.c.src:2754:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3118     2754 |         if (ret->obmeta.base == -1) {\n     3119          |                              ^~\n     3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     103     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 104     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 108     collect2: error: ld returned 1 exit status",
    ">> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 108     collect2: error: ld returned 1 exit status\n     109     failure.\n     110     removing: _configtest.c _configtest.o\n     111     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     112",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     159     gcc: _configtest.c\n>> 160     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     161         1 | #include <sys/endian.h>\n     162           |          ^~~~~~~~~~~~~~\n     163     compilation terminated.\n  >> 164     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "159     gcc: _configtest.c\n  >> 160     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     161         1 | #include <sys/endian.h>\n     162           |          ^~~~~~~~~~~~~~\n     163     compilation terminated.\n>> 164     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     165         1 | #include <sys/endian.h>\n     166           |          ^~~~~~~~~~~~~~\n     167     compilation terminated.\n     168     failure.\n     169     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     275     gcc: _configtest.c\n     276     _configtest.c: In function 'main':\n>> 277     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     278         7 |     (void) SIZEOF_LONGDOUBLE;\n     279           |            ^~~~~~~~~~~~~~~~~\n     280           |            SIZEOF_LONG_DOUBLE\n     281     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "278         7 |     (void) SIZEOF_LONGDOUBLE;\n     279           |            ^~~~~~~~~~~~~~~~~\n     280           |            SIZEOF_LONG_DOUBLE\n     281     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     282     _configtest.c: In function 'main':\n>> 283     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     284         7 |     (void) SIZEOF_LONGDOUBLE;\n     285           |            ^~~~~~~~~~~~~~~~~\n     286           |            SIZEOF_LONG_DOUBLE\n     287     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     421     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 422     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 423     collect2: error: ld returned 1 exit status\n     424     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 425     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 426     collect2: error: ld returned 1 exit status",
    ">> 423     collect2: error: ld returned 1 exit status\n     424     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 425     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 426     collect2: error: ld returned 1 exit status\n     427     failure.\n     428     removing: _configtest.c _configtest.o\n     429     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     430",
    "834     gcc: _configtest.c\n     835     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     836         3 | #pragma clang diagnostic error \"-Wattributes\"\n     837           |\n     838     _configtest.c: In function '__declspec':\n>> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n  >> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {",
    "837           |\n     838     _configtest.c: In function '__declspec':\n  >> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n>> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {\n     844           | ^\n     845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;",
    "843         9 | {\n     844           | ^\n     845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n>> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |\n     851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |",
    "853           |\n     854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"\n     856           |\n     857     _configtest.c: In function '__declspec':\n>> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n  >> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {",
    "856           |\n     857     _configtest.c: In function '__declspec':\n  >> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n>> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {\n     863           | ^\n     864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;",
    "862         9 | {\n     863           | ^\n     864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n>> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |\n     870     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     871        11 | }\n     872           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     1200    gcc: _configtest.c\n     1201    _configtest.c: In function 'main':\n>> 1202    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1203        8 |     (void) HAVE_DECL_SIGNBIT;\n     1204          |            ^~~~~~~~~~~~~~~~~\n     1205          |            HAVE_DECL_ISNAN\n     1206    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1203        8 |     (void) HAVE_DECL_SIGNBIT;\n     1204          |            ^~~~~~~~~~~~~~~~~\n     1205          |            HAVE_DECL_ISNAN\n     1206    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1207    _configtest.c: In function 'main':\n>> 1208    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1209        8 |     (void) HAVE_DECL_SIGNBIT;\n     1210          |            ^~~~~~~~~~~~~~~~~\n     1211          |            HAVE_DECL_ISNAN\n     1212    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1796    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1797    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1798    collect2: error: ld returned 1 exit status\n     1799    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1800    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1801    collect2: error: ld returned 1 exit status",
    ">> 1798    collect2: error: ld returned 1 exit status\n     1799    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1800    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1801    collect2: error: ld returned 1 exit status\n     1802    failure.\n     1803    removing: _configtest.c _configtest.o\n     1804    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1805",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2951     2754 |         if (ret->obmeta.base == -1) {\n     2952          |                              ^~\n     2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2954    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2954    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n>> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n>> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2962          |                         ^\n>> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2965          |                             ^\n  >> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2965          |                             ^\n>> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n  >> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2965          |                             ^\n  >> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n>> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2971          |                          ^\n  >> 2972    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n  >> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2971          |                          ^\n>> 2972    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2974          |                              ^\n     2975    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3118     2754 |         if (ret->obmeta.base == -1) {\n     3119          |                              ^~\n     3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3121    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3121    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n>> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n>> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3129          |                         ^\n>> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3132          |                             ^\n  >> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3132          |                             ^\n>> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n  >> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3132          |                             ^\n  >> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n>> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3138          |                          ^\n  >> 3139    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3140     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n  >> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3138          |                          ^\n>> 3139    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3140     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3141          |                              ^\n  >> 3142    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     834     gcc: _configtest.c\n835     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     836         3 | #pragma clang diagnostic error \"-Wattributes\"\n     837           |\n     838     _configtest.c: In function '__declspec':\n  >> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n  >> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {\n     844           | ^\n845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n  >> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |",
    "implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n  >> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |\n851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |\n     854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "849        11 | }\n     850           |\n     851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |\n854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"\n     856           |\n     857     _configtest.c: In function '__declspec':\n  >> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n  >> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {\n     863           | ^\n864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n  >> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |",
    "implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n  >> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |\n870     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     871        11 | }\n     872           |\n     873     failure.\n\n     ...",
    "...\n\n     2948          |                              ^~\n     2949    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2950    numpy/core/src/multiarray/scalartypes.c.src:2754:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2951     2754 |         if (ret->obmeta.base == -1) {\n     2952          |                              ^~\n     2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2974          |                              ^\n     2975    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2976    numpy/core/src/multiarray/scalartypes.c.src:637:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2977      637 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2978          |     ^~\n\n     ...",
    "...\n\n     3115          |                              ^~\n     3116    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3117    numpy/core/src/multiarray/scalartypes.c.src:2754:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3118     2754 |         if (ret->obmeta.base == -1) {\n     3119          |                              ^~\n     3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     103     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 104     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 108     collect2: error: ld returned 1 exit status",
    ">> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 108     collect2: error: ld returned 1 exit status\n     109     failure.\n     110     removing: _configtest.c _configtest.o\n     111     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     112",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     159     gcc: _configtest.c\n>> 160     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     161         1 | #include <sys/endian.h>\n     162           |          ^~~~~~~~~~~~~~\n     163     compilation terminated.\n  >> 164     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "159     gcc: _configtest.c\n  >> 160     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     161         1 | #include <sys/endian.h>\n     162           |          ^~~~~~~~~~~~~~\n     163     compilation terminated.\n>> 164     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     165         1 | #include <sys/endian.h>\n     166           |          ^~~~~~~~~~~~~~\n     167     compilation terminated.\n     168     failure.\n     169     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     275     gcc: _configtest.c\n     276     _configtest.c: In function 'main':\n>> 277     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     278         7 |     (void) SIZEOF_LONGDOUBLE;\n     279           |            ^~~~~~~~~~~~~~~~~\n     280           |            SIZEOF_LONG_DOUBLE\n     281     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "278         7 |     (void) SIZEOF_LONGDOUBLE;\n     279           |            ^~~~~~~~~~~~~~~~~\n     280           |            SIZEOF_LONG_DOUBLE\n     281     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     282     _configtest.c: In function 'main':\n>> 283     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     284         7 |     (void) SIZEOF_LONGDOUBLE;\n     285           |            ^~~~~~~~~~~~~~~~~\n     286           |            SIZEOF_LONG_DOUBLE\n     287     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     421     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 422     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 423     collect2: error: ld returned 1 exit status\n     424     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 425     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 426     collect2: error: ld returned 1 exit status",
    ">> 423     collect2: error: ld returned 1 exit status\n     424     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 425     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 426     collect2: error: ld returned 1 exit status\n     427     failure.\n     428     removing: _configtest.c _configtest.o\n     429     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     430",
    "834     gcc: _configtest.c\n     835     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     836         3 | #pragma clang diagnostic error \"-Wattributes\"\n     837           |\n     838     _configtest.c: In function '__declspec':\n>> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n  >> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {",
    "837           |\n     838     _configtest.c: In function '__declspec':\n  >> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n>> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {\n     844           | ^\n     845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;",
    "843         9 | {\n     844           | ^\n     845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n>> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |\n     851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |",
    "853           |\n     854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"\n     856           |\n     857     _configtest.c: In function '__declspec':\n>> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n  >> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {",
    "856           |\n     857     _configtest.c: In function '__declspec':\n  >> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n>> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {\n     863           | ^\n     864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;",
    "862         9 | {\n     863           | ^\n     864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n>> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |\n     870     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     871        11 | }\n     872           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     1200    gcc: _configtest.c\n     1201    _configtest.c: In function 'main':\n>> 1202    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1203        8 |     (void) HAVE_DECL_SIGNBIT;\n     1204          |            ^~~~~~~~~~~~~~~~~\n     1205          |            HAVE_DECL_ISNAN\n     1206    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1203        8 |     (void) HAVE_DECL_SIGNBIT;\n     1204          |            ^~~~~~~~~~~~~~~~~\n     1205          |            HAVE_DECL_ISNAN\n     1206    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1207    _configtest.c: In function 'main':\n>> 1208    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1209        8 |     (void) HAVE_DECL_SIGNBIT;\n     1210          |            ^~~~~~~~~~~~~~~~~\n     1211          |            HAVE_DECL_ISNAN\n     1212    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1796    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1797    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1798    collect2: error: ld returned 1 exit status\n     1799    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1800    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1801    collect2: error: ld returned 1 exit status",
    ">> 1798    collect2: error: ld returned 1 exit status\n     1799    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1800    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1801    collect2: error: ld returned 1 exit status\n     1802    failure.\n     1803    removing: _configtest.c _configtest.o\n     1804    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1805",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2951     2754 |         if (ret->obmeta.base == -1) {\n     2952          |                              ^~\n     2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2954    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2954    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n>> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n>> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2962          |                         ^\n>> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2965          |                             ^\n  >> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2965          |                             ^\n>> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n  >> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2965          |                             ^\n  >> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n>> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2971          |                          ^\n  >> 2972    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n  >> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2971          |                          ^\n>> 2972    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2974          |                              ^\n     2975    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3118     2754 |         if (ret->obmeta.base == -1) {\n     3119          |                              ^~\n     3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3121    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3121    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n>> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n>> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3129          |                         ^\n>> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3132          |                             ^\n  >> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3132          |                             ^\n>> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n  >> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3132          |                             ^\n  >> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n>> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3138          |                          ^\n  >> 3139    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3140     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n  >> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3138          |                          ^\n>> 3139    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3140     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3141          |                              ^\n  >> 3142    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -",
    "st.o -o _configtest\n     103     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 104     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 108     collect2: error: ld returned 1 exit status",
    ">> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 108     collect2: error: ld returned 1 exit status\n     109     failure.\n     110     removing: _configtest.c _configtest.o\n     111     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     112",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     159     gcc: _configtest.c\n>> 160     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     161         1 | #include <sys/endian.h>\n     162           |          ^~~~~~~~~~~~~~\n     163     compilation terminated.\n  >> 164     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "159     gcc: _configtest.c\n  >> 160     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     161         1 | #include <sys/endian.h>\n     162           |          ^~~~~~~~~~~~~~\n     163     compilation terminated.\n>> 164     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     165         1 | #include <sys/endian.h>\n     166           |          ^~~~~~~~~~~~~~\n     167     compilation terminated.\n     168     failure.\n     169     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     275     gcc: _configtest.c\n     276     _configtest.c: In function 'main':\n>> 277     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     278         7 |     (void) SIZEOF_LONGDOUBLE;\n     279           |            ^~~~~~~~~~~~~~~~~\n     280           |            SIZEOF_LONG_DOUBLE\n     281     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "278         7 |     (void) SIZEOF_LONGDOUBLE;\n     279           |            ^~~~~~~~~~~~~~~~~\n     280           |            SIZEOF_LONG_DOUBLE\n     281     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     282     _configtest.c: In function 'main':\n>> 283     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     284         7 |     (void) SIZEOF_LONGDOUBLE;\n     285           |            ^~~~~~~~~~~~~~~~~\n     286           |            SIZEOF_LONG_DOUBLE\n     287     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     421     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 422     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 423     collect2: error: ld returned 1 exit status\n     424     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 425     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 426     collect2: error: ld returned 1 exit status",
    ">> 423     collect2: error: ld returned 1 exit status\n     424     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 425     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 426     collect2: error: ld returned 1 exit status\n     427     failure.\n     428     removing: _configtest.c _configtest.o\n     429     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     430",
    "834     gcc: _configtest.c\n     835     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     836         3 | #pragma clang diagnostic error \"-Wattributes\"\n     837           |\n     838     _configtest.c: In function '__declspec':\n>> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n  >> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {",
    "837           |\n     838     _configtest.c: In function '__declspec':\n  >> 839     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     840         5 | int __declspec(thread) foo;\n     841           |                        ^~~\n>> 842     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     843         9 | {\n     844           | ^\n     845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;",
    "843         9 | {\n     844           | ^\n     845     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     846         5 | int __declspec(thread) foo;\n     847           |     ^~~~~~~~~~\n>> 848     _configtest.c:11: error: expected '{' at end of input\n     849        11 | }\n     850           |\n     851     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     852        11 | }\n     853           |",
    "853           |\n     854     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     855         3 | #pragma clang diagnostic error \"-Wattributes\"\n     856           |\n     857     _configtest.c: In function '__declspec':\n>> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n  >> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {",
    "856           |\n     857     _configtest.c: In function '__declspec':\n  >> 858     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     859         5 | int __declspec(thread) foo;\n     860           |                        ^~~\n>> 861     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     862         9 | {\n     863           | ^\n     864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;",
    "862         9 | {\n     863           | ^\n     864     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     865         5 | int __declspec(thread) foo;\n     866           |     ^~~~~~~~~~\n>> 867     _configtest.c:11: error: expected '{' at end of input\n     868        11 | }\n     869           |\n     870     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     871        11 | }\n     872           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     1200    gcc: _configtest.c\n     1201    _configtest.c: In function 'main':\n>> 1202    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1203        8 |     (void) HAVE_DECL_SIGNBIT;\n     1204          |            ^~~~~~~~~~~~~~~~~\n     1205          |            HAVE_DECL_ISNAN\n     1206    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1203        8 |     (void) HAVE_DECL_SIGNBIT;\n     1204          |            ^~~~~~~~~~~~~~~~~\n     1205          |            HAVE_DECL_ISNAN\n     1206    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1207    _configtest.c: In function 'main':\n>> 1208    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1209        8 |     (void) HAVE_DECL_SIGNBIT;\n     1210          |            ^~~~~~~~~~~~~~~~~\n     1211          |            HAVE_DECL_ISNAN\n     1212    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1796    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1797    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1798    collect2: error: ld returned 1 exit status\n     1799    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1800    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1801    collect2: error: ld returned 1 exit status",
    ">> 1798    collect2: error: ld returned 1 exit status\n     1799    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1800    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.12.0-gqoo3ej2ps6t\n             ieh2w4mhwsngpikujxrl/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1801    collect2: error: ld returned 1 exit status\n     1802    failure.\n     1803    removing: _configtest.c _configtest.o\n     1804    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1805",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2951     2754 |         if (ret->obmeta.base == -1) {\n     2952          |                              ^~\n     2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2954    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2953    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2954    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n>> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2955     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2956          |                       ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n>> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2958     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2959          |                        ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2962          |                         ^\n>> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2965          |                             ^\n  >> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2965          |                             ^\n>> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n  >> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2965          |                             ^\n  >> 2966    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n>> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2971          |                          ^\n  >> 2972    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2967     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2968          |                         ^\n  >> 2969    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2970     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2971          |                          ^\n>> 2972    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2973     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2974          |                              ^\n     2975    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3118     2754 |         if (ret->obmeta.base == -1) {\n     3119          |                              ^~\n     3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3121    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3120    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3121    numpy/core/src/multiarray/scalartypes.c.src:4233:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n>> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3122     4233 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3123          |                       ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4234:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n>> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3125     4234 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3126          |                        ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4235:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4235 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3129          |                         ^\n>> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3132          |                             ^\n  >> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4236:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4236 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3132          |                             ^\n>> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n  >> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3132          |                             ^\n  >> 3133    numpy/core/src/multiarray/scalartypes.c.src:4238:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n>> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3138          |                          ^\n  >> 3139    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3140     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3134     4238 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3135          |                         ^\n  >> 3136    numpy/core/src/multiarray/scalartypes.c.src:4239:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3137     4239 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3138          |                          ^\n>> 3139    numpy/core/src/multiarray/scalartypes.c.src:4240:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3140     4240 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3141          |                              ^\n  >> 3142    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -"
]